thing.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
```

Comment:
What's the purpose of this file? Shouldn't the web app front end do this? Is this another test script/kind of integration test?